src/index.html

```diff
@@ -1,14 +1,78 @@
 <!doctype html>
 <html lang="en">
+
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta class="nav__link" charset="UTF-8">
+    <meta class="nav__link" name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+    class="nav__link"
+    http-equiv="X-UA-Compatible"
+    content="ie=edge"
+    >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a class="link" href="#home">
+        <img
+        class="image"
+        src="./images/logo.png"
+        alt="image">
+       </img>
```

Comment:
img does not require a closing tag
```suggestion
        <img
          class="image"
          src="./images/logo.png"
          alt="image"
        >

```